dss/__init__.py

```diff
@@ -13,9 +13,19 @@
 
 from .config import BucketStage, Config
 
-# Constants common to the indexer and query route.
+# CONSTANTS COMMON TO THE INDEXER AND QUERY ROUTE.
+
+# ES index containing all docs
 DSS_ELASTICSEARCH_INDEX_NAME = "hca"
-DSS_ELASTICSEARCH_DOC_TYPE = "hca"
+# ES type within DSS_ELASTICSEARCH_INDEX_NAME with docs
+DSS_ELASTICSEARCH_DOC_TYPE = "docs"
+# ES type within DSS_ELASTICSEARCH_INDEX_NAME with percolate queries
+DSS_ELASTICSEARCH_QUERIES_TYPE = "queries"
+
+# ES index with all registered percolate queries
+DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME = "subscriptions"
+# ES type in DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME with subscriptions
+DSS_ELASTICSEARCH_SUBSCRIPTION_TYPE = "subscription_type"
```

Comment:
I prefer 'subscription' to 'subscription_type'. Alternately, we could omit the subscription doc type and just place them in the top-level index. Yet, use of a doc type is both consistent and provides more flexibility in the future.